tool_collections/cufflinks/gffread/gffread.xml

```diff
@@ -0,0 +1,425 @@
+        #end if
+        #if $gffs.gff_fmt == 'gff':
+            #if $input.datatype.file_ext == 'gft':
+                $gffs.ensembl
+            #end if
+            $gffs.output_cmd
+        #elif $gffs.gff_fmt == 'gtf':
+            $gffs.output_cmd
+        #end if
+    #end if
+]]>
+    </command>
+    <inputs>
+        <param name="input" type="data" format="gff3,gtf" label="Input GFF3 or GTF feature file"/>
+        <!-- filtering -->
+        <param name="filtering" type="select" display="checkboxes" multiple="true" label="filters">
+            <option value="-U">(-U) discard single-exon transcripts</option>
+            <option value="-C">(-C) coding only: discard mRNAs that have no CDS feature</option>
+            <option value="-G">(-G) only parse additional exon attributes from the first exon and move them to the mRNA level (useful for GTF input)</option>
+            <option value="-O">(-O) process also non-transcript GFF records (by default non-transcript records are ignored)</option>
+            <option value="--no-pseudo">(--no-pseudo) filter out records matching the 'pseudo' keyword</option>
+        </param> 
+        <conditional name="region">
+            <param name="region_filter" type="select" label="Filter by genome region">
+                <option value="none">No</option>
+                <option value="filter">Yes</option>
+            </param>
+            <when value="none"/>
+            <when value="filter">
+                <param name="range" type="text" value="" size="60" label="Only show transcripts overlapping coordinate range"
+                    help="-r [['strand']'chr':]'start'..'end' &lt;br&gt; examples: &lt;br&gt; 1000..500000 &lt;br&gt; chr1:1000..500000 &lt;br&gt; +chr1:1000..500000 &lt;br&gt; -chr1:1000..500000" >
+                    <validator type="regex">(([+-])?(\w+:))?\d+\.\.\d+</validator>
+                </param>
+                <param name="discard_partial" type="boolean" truevalue="-R" falsevalue="" check="false" 
+                       label="(-R) and discard all transcripts that are not fully contained within the given range"/>
+            </when>
+        </conditional>
+        <param name="maxintron" type="integer" value="" optional="true" min="0" label="(-i) max_intron - Filter out transcipts with large introns" 
+               help="If set, discard transcripts having an intron larger"/>
+        <param name="chr_replace" type="data" format="tabular" optional="true" label="Replace reference sequence names (e.g. chr1 with 1)" >
+            <help>(-m chr_replace) &lt;br&gt; 
+                  chr_replace is input file is a 2 column tab-delimited file containing a reference (genomic) sequence replacement table with this format: &lt;br&gt; 
+                  "original_ref_ID"  "new_ref_ID" &lt;br&gt; 
+                  GFF records on reference sequences that are not found among the "original_ref_ID" entries in this file will be filtered out
+            </help>
```

Comment:
Ha, this is a nice idea for large help texts. We can also use the CDATA tag here as well.